system_modules/napaudio/src/audio/core/audionodemanager.cpp

```diff
@@ -107,78 +132,77 @@ namespace nap
 			mInputBuffer.resize(inputChannelCount);
 			mChannelCountChangedSignal(*this);
 		}
-		
-		
+
+
 		void NodeManager::setOutputChannelCount(int outputChannelCount)
 		{
 			mOutputChannelCount = outputChannelCount;
 			mOutputMapping.clear();
 			mOutputMapping.resize(mOutputChannelCount);
 			mChannelCountChangedSignal(*this);
 		}
-		
-		
+
+
 		void NodeManager::setInternalBufferSize(int size)
 		{
 			mInternalBufferSize = size;
-			for (auto& node : mNodes)
-				node->setBufferSize(size);
+			for (auto& process : mProcesses)
+				process->setBufferSize(size);
 		}
-		
-		
+
+
 		void NodeManager::setSampleRate(float sampleRate)
 		{
 			mSampleRate = sampleRate;
 			mSamplesPerMillisecond = sampleRate / 1000.;
-			for (auto& node : mNodes)
-				node->setSampleRate(sampleRate);
+			for (auto& process : mProcesses)
+				process->setSampleRate(sampleRate);
 		}
-		
-		
-		void NodeManager::registerNode(Node& node)
+
+
+		void NodeManager::registerProcess(Process& process)
```

Comment:
Purely semantic, but shouldn't `NodeManager` not be renamed to `ProcessManager` ? Doesn't that make more sense ?

Reply:
I agree with you, however this would be a VERY breaking change for a lot of client-code.

Reply:
ok